src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/Dac/Utils/DacPropertySymbolUtils.cs

```diff
@@ -0,0 +1,273 @@
+﻿using Acuminator.Utilities.Common;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+
+namespace Acuminator.Utilities.Roslyn.Semantic.Dac
+{
+	public static class DacPropertySymbolUtils
+	{
+		/// <summary>
+		/// A delegate type for a DAC property which extracts info DTOs about DAC properties/fields from <paramref name="dacOrDacExtension"/> 
+		/// and adds them to the <paramref name="dacInfos"/> collection with a consideration for DAC properties/fields declaration order.
+		/// Returns the number following the last assigned declaration order.
+		/// </summary>
+		/// <typeparam name="TInfo">Generic type parameter representing overridable info type.</typeparam>
+		/// <param name="dacPropertyInfos">The DAC property infos.</param>
+		/// <param name="dacOrDacExtension">The DAC or DAC extension.</param>
+		/// <param name="startingOrder">The declaration order which should be assigned to the first DTO.</param>
+		/// <returns/>
+		private delegate int AddDacPropertyInfoWithOrderDelegate<TInfo>(OverridableItemsCollection<TInfo> dacPropertyInfos,
+																		ITypeSymbol dacOrDacExtension, int startingOrder)
+		where TInfo : IOverridableItem<TInfo>;
+
+
+		#region Dac Properties
```

Comment:
I propose to remove this region and split file if it is not readable without the region

Reply:
I agree